src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -121,6 +121,9 @@ public function render_dropdowns() {
 		}
 
 		?><script type="text/template" id="tmpl-tribe-select-<?php echo esc_attr( $this->post_type ); ?>"><?php $this->single_post_dropdown( 0 ); ?></script><?php
+		
+		// @todo: add more check here for the context: when/where should we do this?
```

Comment:
This feels like the right place/time to do this to me. I can't think of any context where this _shouldn't_ be executed.